docs/user/aws/README.md

```diff
@@ -0,0 +1,22 @@
+# AWS Account Set Up for OpenShift
+
+This document is a guide for preparing any new AWS account for use with OpenShift. It 
+will help prepare an account to create a single cluster and provide insight for adjustments which may be 
+needed beyond.
+
+Follow along with the steps & links below to configure your AWS cluster:
+
+1. [Route53](route53.md)
+2. [AWS Limits](limits.md)
+3. [IAM User](iam.md)
+4. [AWS CLI](cli.md)
+5. [OpenShift Installation](install.md)
+6. [IAM User: Revisited](iam_after.md)
+
+## Reporting Issues
+
+Please see the [Issue Tracker][issues_aws] for current known issues.
+Please report a new issue if you do not find an issue related to any trouble
+you’re having.
```

Comment:
Copy/paste artifact from Google Docs. Fixed. Thanks!

Reply:
There are still a few curly quotes left:

```console
$ git describe origin/pr/1030
v0.9.0-master-13-g55ebc12
$ git show origin/pr/1030 | grep '’\|“\|”'
+future date so a specific policy can be created and attached. Until then, attach the predefined “AdministratorAccess” 
+Step 3 is optional and we’ll skip it.
+Step 4 allows us to review the settings we’ve selected. Make sure your screen reflects your chosen name and 
+Below, we’ll identify OpenShift cluster needs and how those impact some of those limits.
+“EC2-VPC Elastic IPs”. The installer creates a public and private subnet for each 
+account’s default limit. If you intend to start with a higher number of workers, enable autoscaling and large workloads 
+or a different instance type, please ensure you have the necessary remaining instance count within the instance type’s 
+Amazon’s Route53 service is used by the OpenShift installer to configure cluster DNS resolution and provide name lookup 
+the same account as your OpenShift cluster. You must also ensure the zone is “authoritative” for the domain. There are 
+two ways to do this outlined below: root domain and subdomain. A root domain is “openshiftcorp.com”. A subdomain is of 
+the form “clusters.openshiftcorp.com”.
+To use the root domain, you’d create the hosted zone with the value “openshiftcorp.com”. To use a subdomain, you’d 
+create a hosted zone with the value “clusters.openshiftcorp.com”. (Use appropriate domain values for your situation.)
+For either a root domain “openshiftcorp.com” or a subdomain “clusters.openshiftcorp.com”, you must extract the new 
+parent/root domain. This may require a request to your company’s IT department or the division which controls the root 
```